api/v1/openapi.yaml

```diff
@@ -0,0 +1,953 @@
+---
+swagger: '2.0'
+info:
+  title: Cilium API
+  description: Cilium
+  version: v1beta
+schemes:
+- http
+x-schemes:
+- unix
+basePath: "/v1beta"
+produces:
+- application/json
+consumes:
+- application/json
+paths:
+  "/healthz":
+    get:
+      summary: Retrieve health of Cilium daemon
+      description: |
+        Returns the health and status of the Cilium daemon and all related
+        components.
+      tags:
+      - daemon
+      responses:
+        '200':
+          description: Success
+          schema:
+            "$ref": "#/definitions/StatusResponse"
+  "/config":
+    get:
+      summary: Retrieve daemon configuration
+      description: |
+        Returns the configuration of the daemon which includes addressing
+        and configuration settings.
+      tags:
+      - daemon
+      responses:
+        '200':
+          description: Success
+          schema:
+            "$ref": "#/definitions/DaemonConfigurationResponse"
+    put:
+      summary: Update daemon configuration
+      description: |
+        Updates the daemon configuration with the provided ConfigurationMap
+        and regenerates & recompiles all required datapath components.
+      tags:
+      - daemon
+      parameters:
+      - name: configuration
+        in: body
+        required: true
+        schema:
+          "$ref": "#/definitions/ConfigurationMap"
+      responses:
+        '200':
+          description: Success
+        '400':
+          description: Bad configuration parameters
+          schema:
+            "$ref": "#/definitions/Error"
+        '500':
+          description: Recompilation failed
+          x-go-name: Failure
+          schema:
+            "$ref": "#/definitions/Error"
+  "/endpoint/{id}":
+    get:
+      summary: Retrieve an endpoint
+      description: |
+        Looks up and returns the endpoint described with the endpoint ID.
+      tags:
+      - endpoint
+      parameters:
+      - "$ref": "#/parameters/endpoint-id"
+      responses:
+        '200':
+          description: Success
+          schema:
+            "$ref": "#/definitions/Endpoint"
+        '400':
+          description: Invalid endpoint ID format for specified type
+          x-go-name: Invalid
+          schema:
+            "$ref": "#/definitions/Error"
+        '404':
+          description: Endpoint not found
+    delete:
+      summary: Delete endpoint
+      description: |
+        Deletes the endpoint specified by the ID. Deletion is imminent and
```

Comment:
How does 3rd parties know there's a "force" option available by only reading the docs?